tests/rt.sh

```diff
@@ -1201,7 +1238,11 @@ while read -r line || [[ -n "${line}" ]]; do
       fi
     fi
 
-    create_or_run_compile_task
+    if [[ ${RUN_ONLY} == true ]]; then
```

Comment:
this is something i have wanted to see for a long time.

is this going to enable users to run the ufs-wm 'run' step on an existing fv3.exe to just check if the run will pass (without having to wait for a re-compile)?

Reply:
That is the idea, but it isn't working yet, unfortunately.